cg/meta/deliver.py

```diff
@@ -49,14 +52,14 @@ def __init__(
         self.skip_missing_bundle = self.delivery_type in constants.SKIP_MISSING
 
     def set_dry_run(self, dry_run: bool) -> None:
-        """Update dry run"""
+        """Update dry run."""
         LOG.info("Set dry run to %s", dry_run)
         self.dry_run = dry_run
 
     def deliver_files(self, case_obj: Family):
-        """Deliver all files for a case
+        """Deliver all files for a case.
 
-        If there are sample tags deliver all files for the samples as well
+        If there are sample tags deliver all files for the samples as well.
         """
         case_id: str = case_obj.internal_id
         case_name: str = case_obj.name
```

Comment:
One more thing! I know it not part of you PR, but I think we can rewrite the if/elif below (L68-71) to:
```
        if not last_version:
            if not self.case_tags:
                LOG.info("Could not find any version for {}".format(case_id))
            elif self.skip_missing_bundle == False:
                raise SyntaxError("Could not find any version for {}".format(case_id))
```
It is more readable and should be equivalent